source/fab/util.py

```diff
@@ -250,10 +222,12 @@ def run_command(command, env=None):
 
     """
     logger.debug(f'run_command: {command}')
-    res = subprocess.run(command, capture_output=True, env=env)
+    res = subprocess.run(command, capture_output=True, env=env, cwd=cwd)
     if res.returncode != 0:
         raise RuntimeError(f"Command failed:\n{command}\n{res.stderr.decode()}")
 
+    return res.stdout
```

Comment:
This may be OK depending on what you run, but users won't be able to see the stdout on error, which may contain clues of why it fails.

Reply:
Good point. I'll add it to the exception.